addons/hr/tests/test_mail_activity_plan.py

```diff
@@ -195,9 +195,10 @@ def test_responsible(self):
             # Happy case
             form = self._instantiate_activity_schedule_wizard(employees)
             form.plan_id = self.plan_onboarding
-            self.assertEqual(form.plan_assignation_summary,
-                             '<ul><li>To-Do: Plan training</li><li>To-Do: Training</li>'
-                             '<li>To-Do: Send feedback to the manager</li></ul>')
+            self.assertIn('Plan training', form.plan_summary)
+            self.assertIn('Training', form.plan_summary)
+            self.assertIn('Send feedback to the manager', form.plan_summary)
+            self.assertEqual(form.plan_summary.count('</li>'), 3)
```

Comment:
Not sure about counting the tags, can't we assert a little more here above to make sure every line is in its own tag if we want to assert the tags?